underscore.js

```diff
@@ -647,6 +647,23 @@
     return range;
   };
 
+  // Split an **array** into several arrays containing **count** or less elements
+  // of initial array
+  _.chunk = function(array, count) {
+    count = count || 1;
```

Comment:
I'm actually leaning with @megawac here. We're not cutting the array into 3 slices, we're gathering into groups of 3. And groups of 0 don't have anything.